### PR TITLE
add the DAG version of the siamese model prototext and

### PR DESCRIPTION
**Pull request by JaeseungYeom**
that of the prototext for fine-tuning

The differences from the originial prototext are
- batch normalization weights are frozen
- batch normalization weights are transferred for fine-tuning

### CODE REVIEW

File: model_zoo/models/siamese/finetune-cub/model_cub_batchnorm_transferred_and_frozen.prototext [hunk 1]
```diff
@@ -0,0 +1,944 @@
...
+
+  weights {
+    name: "bn_conv1_bias"
+    constant_initializer {
+      value: 0.0
+    }
+  }
+
+
+  weights {
```
Comment (timmoon10) [inline model_zoo/models/siamese/finetune-cub/model_cub_batchnorm_transferred_and_frozen.prototext]: I think it would be safer to initialize running statistics weights with no optimizer:
```
weights {
  name: "something"
  some_initializer { something }
  optimizer {}
}
```
It shouldn't have an effect since we don't provide gradient updates to the running statistics, but it makes our usage clearer.

**Comment by JaeseungYeom** [comment]
addressed Tim's suggestion.
